frontend/src/components/MapView/Layers/AdminLevelDataLayer/index.tsx

```diff
@@ -66,7 +73,7 @@ export const addFillPatternImageInMap = (
     const imageId = `fill-pattern-${layer.id}-legend-${index}`;
     if (!map.hasImage(imageId)) {
       // Add the image since it doesn't exist
-      map.addImage(imageId, image);
+      map.addImage(imageId, image, { pixelRatio: 4 });
```

Comment:
We might need to test on standard screens to make sure it's still ok (macbooks have an internal pixelRatio of 2 compared to the default 1) so it might affect the output of the image here.